src/platforms/rcore_android.c

```diff
@@ -1225,7 +1225,8 @@ static int32_t AndroidInputCallback(struct android_app *app, AInputEvent *event)
             {
                 CORE.Input.Keyboard.currentKeyState[key] = 1;   // Key down
 
-                CORE.Input.Keyboard.keyPressedQueue[CORE.Input.Keyboard.keyPressedQueueCount] = key;
+                CORE.Input.Keyboard.keyPressedQueue[CORE.Input.Keyboard.keyPressedQueueCount].keycode = key;
+                CORE.Input.Keyboard.keyPressedQueue[CORE.Input.Keyboard.keyPressedQueueCount].scancode = -1;
```

Comment:
It seems scancode can be read with NDK [AKeyEvent_getScanCode()](https://developer.android.com/ndk/reference/group/input#group___input_1ga4a0a846b7a195aeb290dfcd2250137d9).